cli/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 _Released 1/16/2024 (PENDING)_
 
 **Bugfixes:**
-
+- Force `moduleResolution` to `node` when `typescript` projects are detected to correctly run Cypress. This change should not have a large impact as `commonjs` is already forced when `ts-node` is registered. Fixes [#27731](https://github.com/cypress-io/cypress/issues/27731).
```

Comment:
maybe worth noting it doesn't work with the esm loader

Reply:
or maybe that it doesn't impact it?